### ISSUE TITLE
Higurashi Ch1 Onikakushi - HD Original Sprites

### ISSUE THREAD

**Issue by JmTrad** [07th-mod/onikakushi#173]
This patch is meant to upgrade the original Higurashi spirtes for HD versions. I used the waifu2x program for this. Very soon I will bring HD versions of the other 5 games. Hope you like it. ~JmTrad.

https://drive.google.com/file/d/1eDp5lSVE07BGDPL_-8FIo-PrLJcPJ5eu/view?usp=sharing

**Comment by enumag** [comment]
It might be better to publish this on Steam instead. This group is primarily focusing on PS3 patches so this seems a bit out of our scope.

**Comment by JmTrad** [comment]
There are people like me who only apply the voice patch to play with the original sprites. This patch is to enhance the experience of these people. I asked in the discord and they recommended me to post here. But thanks for the tip, I plan to share on Steam as well.

**Comment by enumag** [comment]
Oh, ok. We'll talk about it more on Discord then. I'm not exactly sure what are the plans with the voice-only patches. Thanks for sharing.

**Comment by ItaloKnox** [comment]
Yes, I recommended that. I'll pull these to a new tag and edit the wiki to recommend them. Since we don't have an installer for the voices-only patch (yet), I will just recommend installing it alongside the voices.

I'll leave this open until I do all the work, might take a little time 'cause I'm still recovering from the Tsumi fiesta.
